DopplerJobsServer/Startup.cs

```diff
@@ -95,8 +95,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             app.UseHangfireDashboard("/hangfire", new DashboardOptions
             {
                 PrefixPath = Configuration["PrefixHangfireDashboard"],
-                Authorization = new[] {new HangfireAuthorizationFilter()},
-                IsReadOnlyFunc = context => !env.IsDevelopment()
+                Authorization = new[] {new HangfireAuthorizationFilter()}
```

Comment:
🎉  did you find a way to authorize access in production?

Reply:
nop, it is only temporary